enterprise/api/v1/serializers.py

```diff
@@ -952,3 +952,43 @@ def validate(self, data):  # pylint: disable=arguments-differ
         if not data.get('email') and not data.get('email_csv'):
             raise serializers.ValidationError('Must include either email or email_csv in request.')
         return data
+
+# pylint: disable=abstract-method
+class EnterpriseCustomerBulkSubscriptionEnrollmentsSerializer(serializers.Serializer):
+    """
+    Serializes an email field and for bulk enrollment requests.
+    """
+    emails = serializers.DictField(
```

Comment:
I don't have much experience creating dictionary fields, I'm fairly certain I got this right but would love if someone could confirm that I've got this the way we want.